test/markup/css/variables.txt

```diff
@@ -7,3 +7,9 @@ body {
  --textBlue: blue;
  color: var(--textBlue);
 }
+
+body {
+ --_margin-top: 1rem;
+ margin-top: var(--_margin-top);
+}
+
```

Comment:
You'll also need to update the matching `variables.expect.txt` file with the correct output.